lib/locales/en.yml

```diff
@@ -10,6 +10,7 @@ en:
   exercise: Exercise
   equation: Equation
   chapter: Chapter
+  module: Module
```

Comment:
Hmm :thinking:  I see that actually you are adding `label_text: 'module'` for English books. Can we remove in this case condition that I'm talking about in my previous comment?